File: iniciante/javascript/1040.js
var input = require('fs').readFileSync('stdin','utf8')
var lines = input.split('\n')

let entrada = lines.shift().split(' ')
let nota1 = parseFloat(entrada[0])
let nota2 = parseFloat(entrada[1])
let nota3 = parseFloat(entrada[2])
let nota4 = parseFloat(entrada[3])

const media = function(a, b, c, d){
    return (a * 2 + b * 3 + c * 4 + d * 1)/10
}

let resultado = media(nota1, nota2, nota3, nota4)

console.log(`Media: ${resultado.toFixed(1)}`)

if(resultado >= 7.0){
    console.log('Aluno aprovado.')
}else if(resultado < 5.0){
    console.log('Aluno reprovado.')
}else if(resultado >= 5.0 && resultado <= 6.9){
    console.log('Aluno em exame.')
    let nota5 = parseFloat(lines.shift())
    console.log(`Nota do exame: ${nota5.toFixed(1)}`)
    let mediaFinal = (nota5 + resultado)/2
    if( mediaFinal < 5){
        console.log('Aluno reprovado.')
        console.log(`Media final: ${mediaFinalto.Fixed(1)}`)
    }else{
        console.log('Aluno aprovado.')
        console.log(`Media final: ${mediaFinal.toFixed(1)}`)
    }

}

File: iniciante/python/1040.py
# -*- coding: utf-8 -*-

entrada = input().split(' ')
entrada2 = 0.0

media = (float(entrada[0]) * 2 +
         float(entrada[1]) * 3 +
         float(entrada[2]) * 4 +
         float(entrada[3]) * 1) / 10
media2 = 0.0

print(f'Media: {media:.1f}')
if media >= 7:
    print('Aluno aprovado.')
elif media < 5.0:
    print('Aluno reprovado.')
elif media >= 5.0 and media <= 6.9:
    print('Aluno em exame.')
    entrada2 = float(input())
    print(f'Nota do exame: {entrada2:.1f}')
    media2 = (entrada2 + media) / 2

    if media2 >= 5.0:
        print('Aluno aprovado.')
    else:
        print('Aluno reprovado.')
    print(f'Media final: {media2:.1f}')
